src/components/StatsBoxGrid.tsx

```diff
@@ -233,18 +127,90 @@ const GridItem: React.FC<IGridItemProps> = ({ metric, dir }) => {
   )
 
   return (
-    <Box>
+    <Box
+      position="relative"
+      color="text"
+      height="20rem"
+      background={""}
+      display="flex"
+      flexDirection="column"
+      justifyContent="space-between"
+      alignItems="flex-start"
+      borderX={{
+        base: "0px solid #000000",
+        lg: "1px solid",
+      }}
+      borderY="1px solid"
+      marginTop={{
+        base: "-1px",
+        lg: "0",
+      }}
+      padding={{
+        base: "1rem",
+        lg: "1.5rem",
+      }}
+      paddingTop={{
+        base: "2rem",
+        lg: "0",
```

Comment:
When this is applied with the `lg` value, it still overrides the value coming from the `padding` shorthand prop, so at `lg` and greater breakpoints, there is still `0` top padding.

This should be simplified to just using the shorthand `padding` prop only.
```jsx
padding={{
  base: "2rem 1rem 1rem",
  lg: "1.5rem"
}}
```
> Side note: I personally need to explore why it's not possible to use token values when providing multiple values in the shorthand. I should know this answer! 😅